draft-webtransport-http2.md

```diff
@@ -5,7 +5,7 @@ docname: draft-webtransport-http2-latest
 
```

Comment:
Line 384: "best treated" - maybe this should be a RECOMMENDED?

Reply:
Done